src/backend/aspen/database/models/usergroup.py

```diff
@@ -29,6 +30,14 @@ class Group(idbase, DictMixin):  # type: ignore
     division = Column(String, nullable=True)
     location = Column(String, nullable=True)
 
+    # Default location context (int'l or division or location level)
+    default_tree_location_id = Column(
+        Integer,
+        ForeignKey(Location.id),
+        nullable=True,
+    )
+    default_tree_location = relationship("Location")  # type: ignore
+
```

Comment:
So this associates a group with a default tree location, which is pulled from Lucia's location table and later sent into tree building. In the yaml builder below, we use `if not location.division:` to tell it's a country build, but I was searching through the location table (on Aspen upload page), it looks like all entries have a division. Are those locations w/o division simply not displayed for sample upload location dropdown?

Reply:
You're right! We currently don't have any locations without a division. We have a couple options for that:
- Manually create locations without divisions for each group that needs country-level runs (easy enough)
- Automatically generate country-level locations when we import gisaid data the same way we generate division-level locations.
We can do either or both, and they're both pretty straightforward

Reply:
Route 1 with a lighter lift sounds good to me. I'm slightly unsure where we'd end up w LMIC users b/c their tree building may run into various other problems that  in the end turned out to be less suitable for Aspen.

Reply:
@danrlu what do you mean by "various other problems that in the end turned out to be less suitable for Aspen", can you explain a bit more?